AEPIdentity/Sources/IdentityState.swift

```diff
@@ -252,6 +251,37 @@ class IdentityState {
         lastValidConfig = newConfig
     }
 
+    /// Invoke each time when we receive Analytics Response event
+    /// - Parameters:
+    ///   - event: the event from Analytics Respsonse
+    ///   - eventDispatcher: a function which when invoked dispatches an `Event` to the `EventHub`
+    func handleAnalyticsResponse(event: Event, eventDispatcher: (Event) -> Void) {
+        guard let aid = event.data?[IdentityConstants.Analytics.ANALYTICS_ID] as? String, !aid.isEmpty else {
+            Log.debug(label: "\(LOG_TAG):\(#function)", "Analytics Tracking ID is not found or empty")
+            return
+        }
+
+        if !(identityProperties.isAidSynced ?? false) {
+            // dispatch events
+            let identifiers: [String: String] = [IdentityConstants.EventDataKeys.ANALYTICS_ID: aid]
+            let syncData: [String: Any] = [
+                IdentityConstants.EventDataKeys.IDENTIFIERS: identifiers,
+                IdentityConstants.EventDataKeys.FORCE_SYNC: false,
+                IdentityConstants.EventDataKeys.IS_SYNC_EVENT: true,
+                IdentityConstants.EventDataKeys.AUTHENTICATION_STATE: MobileVisitorAuthenticationState.unknown.rawValue,
+            ]
+
+            let avidEvent = Event(name: IdentityConstants.EventNames.AVID_SYNC_EVENT,
+                                  type: EventType.identity,
+                                  source: EventSource.requestIdentity,
+                                  data: syncData)
+            eventDispatcher(avidEvent)
+            identityProperties.isAidSynced = true
```

Comment:
Let's update identity properties and save to persistence before we dispatch the event.

Reply:
updated